staking/app/StakeConnection.ts

```diff
@@ -597,6 +597,15 @@ export class StakeConnection {
     await this.provider.sendAndConfirm(transaction);
   }
 
+  /**
+   * Locks the specified amount of tokens in governance.
+   */
+  public async signLlc(stakeAccount: StakeAccount) {
+    const transaction: Transaction = new Transaction();
```

Comment:
Man I feel like we should really have a test for this

Reply:
Where someone `depositAndLock` then someones updates the Llc Agreement and then that same someone then uses `signLlc`